### ISSUE TITLE
Make using auto-install prompt better

### ISSUE THREAD

**Issue by PallHaraldsson** [JuliaLang/Pkg.jl#2873]
A.
The new auto-install prompt is great, but you do not get it if you do not already have a .julia folder, when it's maybe most needed (i.e. for newbies). I guess that may be an oversight and easily fixed. Maybe then we'll see it in Julia 1.7.1?

B.
I'm not sure if the auto-install uses the same default as with pkg> add

Does anyone know, and if it doesn't preserve versions, or at least prevent downgrades, then I think that should be the default. Doing otherwise seems too implicit. I would even argue for a new default for pkg> add ...

**Comment by IanButterworth** [comment]
> The new auto-install prompt is great, but you do not get it if you do not already have a .julia folder,

Yeah. There's been some discussion about what best to do. Options are;

1. Auto setup the depot and install the default registries. What https://github.com/JuliaLang/Pkg.jl/pull/2862 did, but was considered too presumptive
2. Prompt to offer to set up the default registries.
3. Warn that the registries need installing. What Pkg master does (although it's not on julia master yet) via https://github.com/JuliaLang/Pkg.jl/pull/2865 but there's questions about the right suggestion to make

For people who specifically don't want a registry installed, an env var could be exposed to suppress any of the above.

> B. I'm not sure if the auto-install uses the same default as with pkg> add

It does the default for `Pkg.add` which I believe is the same as the default for `pkg> add` i.e. tiered.
Perhaps that could be forked off into a different issue rather than discussed here?
